pims/bioformats.py

```diff
@@ -5,32 +5,53 @@
 from pims.base_frames import FramesSequence
 from pims.frame import Frame
 from warnings import warn
-from os.path import isfile
+import os
 
 try:
-    import javabridge
+    import jpype
 except ImportError:
-    javabridge = None
+    jpype = None
 
-try:
-    import bioformats
-except ImportError:
-    bioformats = None
+LOCI_TOOLS_PATH = os.path.join(os.path.dirname(__file__),
```

Comment:
I think that downloading at run time is the right approach (to prevent any strange installation order dependencies (install pims before jtype, you have both, but can't use bioformats)).

It is user hostile even by my standards to ask the users to download and copy files to a location we can't tell them directly.

It may be complicated, but it is the cost of doing business here.